pudb/debugger.py

```diff
@@ -945,6 +945,9 @@ def insert_watch(w, size, key):
         self.var_list.listen("[", partial(change_rhs_box, 'variables', 0, -1))
         self.var_list.listen("]", partial(change_rhs_box, 'variables', 0, 1))
 
+        self.var_list.toggle("j", partial(self.rhs_scroll_down))
+        self.var_list.toggle("k", partial(self.rhs_scroll_up))
```

Comment:
Shouldn't these be `listen`?

Reply:
Yes, that was my fault. I had apparently tested it before reverting the changes to the SignalWrapper.. Just fixed it!